BridgeSDK/BridgeAPI/SBBUploadManager.m

```diff
@@ -255,6 +255,125 @@ - (SBBUploadSession *)uploadSessionForFile:(NSString *)fileURLString
     return uploadSession;
 }
 
+// When uploading a file to Bridge, first we use background download to get an SBBUploadSession object from Bridge.
+// Then we use the pre-signed URL in that SBBUploadSession to upload our file to S3.
+// When that finishes, we tell Bridge that the requested upload is completed.
+// To avoid having multiple layers of nested completion handlers in one method, they're broken out by step:
+
+// --- Here's the completion handler for the step where we tell Bridge the upload completed:
+- (void)toldBridgeFileUploadedWithTask:(NSURLSessionUploadTask *)uploadTask forBridgeSession:(SBBUploadSession *)uploadSession error:(NSError *)error
```

Comment:
well, it's a "handler" that gets called when we've "told" Bridge the file upload completed...